openedx/features/enterprise_support/api.py

```diff
@@ -97,17 +92,31 @@ def consent_required(self, enrollment_exists=False, **kwargs):
         return response['consent_required']
 
 
+class ConsentApiServiceClient(ConsentApiClient):
+    """
+    Class for producing an Enterprise Consent API client with service user.
+    """
+
+    def __init__(self):
```

Comment:
Do you want to abstract this into an `EnterpriseServiceClientMixin`? That way, we can just drop that mixin in front of the relevant base client (`ConsentApiClient` or `EnterpriseApiClient`) in the inheritance order and not even need to do anything in the class body.

Reply:
Your suggestion would work but I think explicitly getting `EnterpriseApiServiceClient` and `ConsentApiServiceClient` make them more readable.

Reply:
Right; I'm saying something like this:

```python
class EnterpriseApiServiceClient(EnterpriseServiceUserClient, EnterpriseApiClient):
    pass
```

Reply:
I have updated the PR with this suggestion. Please have a look.